manifests/install.pp

```diff
@@ -42,6 +42,49 @@
       path    => '/bin'
     }
   }
+
+  elsif $opendaylight::install_method == 'deb'{
+    
+    include apt
+    
+    apt::backports { 'jessie-backports':
+      comment  => 'This is Debian jessie-backports to install Java 8',
+      location => 'http://httpredir.debian.org/debian',
+      release  => 'jessie-backports',
+      repos    => 'main',
+      notify   => Exec['apt_update'],
```

Comment:
Can you push these changes and open a vagrant-opendaylight PR? Also for the docker beaker tests below please - they both look cool. :+1: 

> I don't think this is expected behavior. Am I on the right track?

Yes, everything looks right, very far along the right track.

The only difference I would do is letting the Vagrant Puppet provisioner do the puppet apply, vs you doing it manually. Can force a reprovision `vagrant provision <box>`.

Is ODL installed/running from your `vagrant up` already? (`sudo systemctl status opendaylight`)

> Umm...I haven't tested it completely. Basically this should add jessie-backports.

Okay, I guess it's resolving to that via some puppet magic I'm missing and will see in a log eventually.